docs/artifacts/the-team-is-a-transport.html

```
@@ -0,0 +1,217 @@
+    <span><span class="pill acc">DERIVED</span> computed from measured parts</span>
+    <span><span class="pill grey">INHERITED</span> quoted from an earlier report, not re-run</span>
+  </p>
+
+  <h2>The picture in one line</h2>
+  <p>Two carpenters share a workshop. One designs (Claude), one builds (Codex). The work was never the problem. The problem is how the builder tells the designer "it's done": a <em>shout across the yard</em> that is often not heard, versus a <em>note left on the bench</em> that is always there in the morning.</p>
+
+  <figure>
+    <pre class="mermaid">
+%%{init: {'theme':'base','themeVariables':{'primaryColor':'#FFFFFF','primaryTextColor':'#1B2430','primaryBorderColor':'#7A847E','lineColor':'#1B2430','fontFamily':'IBM Plex Mono, monospace','fontSize':'13px'}}}%%
+flowchart LR
+  A["Claude architect<br/>(this session)"] -->|"spec, via stdin -"| X["Codex lane<br/>(codex exec)"]
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)